HLSL/FastNoiseLite.hlsl

```diff
@@ -47,7 +47,7 @@
 // VERSION: 1.0.1
 // https://github.com/Auburn/FastNoise
 
-// Switch between using floats or doubles for input position
+// [flatten] switch between using floats or doubles for input position
```

Comment:
This looks like an overzealous find and replace

Reply:
You're too good at this.

Reply:
At least it was commented out ;). Let me restore that line.